### PR TITLE
docs: Update FAQ with information on DockerStatic containers

### PR DESCRIPTION
**Pull request by sxa**
Signed-off-by: Stewart X Addison <sxa@redhat.com>

##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [x] [FAQ.md](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

**Review by Haroon-Khel — APPROVED** [review]
LGTM. Couldnt find an error 👍🏻

File: FAQ.md [hunk 1]
```diff
@@ -239,3 +239,47 @@ NOTE ref inventory: If you are adding a new type of machine (`build`, `perf` etc
...
+those values, and we should look at whether we can reasonably autodetect or
+parameterize those values. Potentially we could also scale up and create
+more than one of each OS on a given host. To set up a host for running these
+you can use the
+[dockerhost.yml](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/dockerhost.yml)
+playbook (Also available via AWX).
+
+Once the static docker containers have been created they are connected into
+jenkins with a `test-docker-` prefix. Ideally the description of the machine
+in jenkins should list which host it's on, but can you also look up the
```
Comment (Willsparker) [inline FAQ.md:272]:
```suggestion
in jenkins should list which host it's on, but you can also look up the
```
Mismatched words

**Review by Willsparker — CHANGES_REQUESTED** [review]
Otherwise, LGTM

**Review by karianna — APPROVED** [review]
LGTM

**Review by Willsparker — APPROVED** [review]
LGTM

**Pull request merged**